task/gitlab-jq/0.1/gitlab-jq.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: gitlab-jq
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/categories: Git
+    tekton.dev/tags: gitlab, git
+    tekton.dev/displayName: "Send a request to the Gitlab REST API and filter the response with jq."
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  description: >-
+    This Task sends a request to the Gitlab REST API and filters the JSON response with jq.
+    By defining a jq filter expression, it's possible to select specific fields from the JSON response and format them as needed.
+    This Task stores the output of the jq command as a result.
+  params:
```

Comment:
can you please document the params in README so that so it's more clearer to the users?

Reply:
I just pushed a new README